### PR TITLE
Issue Template for LWKD Newsletter update request 

### PR DESCRIPTION
**Pull request by fykaa**
Please examine the following two points before merging this pr:
1. For Related projects, we need to link a list of projects from which we welcome submissions. Check [issue](https://github.com/kubernetes-sigs/lwkd/issues/232).
2. Is the label 'kind/feature' correct, or should it be changed to 'kind/documentation'?

### CODE REVIEW

File: _posts/2023-09-24-update.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+## KEP of the Week
+
+
+## Other Merges
+*
+*
+
+
+## Promotions
```
Comment (chris-short) [inline _posts/2023-09-24-update.md:29]: Not trying to step on any toes here, but as someone who works with product marketing, some folks will see this as "Promotion" in a "Oh can I buy a spot in this newsletter?" kind of way. Might want to say Software/API/etc. in front of it.

Comment (jberkus) [reply _posts/2023-09-24-update.md]: We've been using that section name for 3 years now.  I don't think this is sufficiently enough of an issue to warrant changing it.

**Comment by jberkus** [comment]
I want to do some editing on this form, will get to it soon

File: .github/ISSUE_TEMPLATE/lwkd-report.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+<!-- Please only use this template for submitting content you'd like to feature in the LWKD Newsletter -->
+<!-- See https://kubernetes.io/docs/contribute/start/ for guidance on writing an actionable issue description. -->
+
+**What would you like to be added**
+<!-- Please provide a detailed description of how this news, feature, or enhancement should function from the reader's perspective -->
+
+
+**Why is this needed**
+<!-- Provide more context of how this information can be used in practice?, who will benefit from this?, why are they being added? -->
+
```
Comment (jberkus) [inline .github/ISSUE_TEMPLATE/lwkd-report.md:18]: These three sections should be collapsed down to:

**Your Item for LWKD**

Please write *one sentence* with your news item for LWKD.    This line should very briefly explain what the item is and why it's important.  

**Links for Your Item**

Share one or two links for this item.  Links to merged pull requests and/or Github issues are better than links to mailing lists or web pages.

File: .github/ISSUE_TEMPLATE/lwkd-report.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+---
+name: Feature an update for LWKD
+about: Submit a news, PR, content or update for the next 'Last Week in Kubernetes Development' Newsletter
+title: 'Add to LWKD - [Title]'
```
Comment (jberkus) [inline .github/ISSUE_TEMPLATE/lwkd-report.md:4]:
```suggestion
title: 'News Item: [Title]'
```

File: .github/ISSUE_TEMPLATE/lwkd-report.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+---
+name: Feature an update for LWKD
```
Comment (jberkus) [inline .github/ISSUE_TEMPLATE/lwkd-report.md:2]:
```suggestion
name: Submit a News Item
```

File: .github/ISSUE_TEMPLATE/lwkd-report.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+**Section:**
+<!-- Choose the most appropriate portion of the newsletter for this information -->
+- [ ] Developer News
+- [ ] Release Schedule
+- [ ] Featured PRs
+- [ ] Other Merges
+- [ ] Version Promotions/ Graduates
+- [ ] Deprecated
+- [ ] Version Updates
+- [ ] Related Projects
```
Comment (jberkus) [inline .github/ISSUE_TEMPLATE/lwkd-report.md:32]:
```suggestion
- [ ] Related Projects
- [ ] Not Sure / Other
```

**Comment by jberkus** [comment]
Thanks!

**Pull request merged**